e107_plugins/news/news_carousel_menu.php

```diff
@@ -127,6 +127,7 @@
 	
 	$footer = str_replace("{NAV}", implode("\n",$nav), $NEWS_MENU_TEMPLATE['carousel']['end']); 
 
-	e107::getRender()->tablerender('',$header.$text.$footer,'news-carousel'); //TODO Tablerender(). 
+	$caption = '';
```

Comment:
`$caption` should become a menu config value so that webmaster will be able to set (or leave blank)  caption for every menu instance.